# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/nr_socket_prsock.cpp
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 media/mtransport/nr_socket_prsock.cpp
--- a/media/mtransport/nr_socket_prsock.cpp	Wed Sep 18 14:27:42 2019 +0000
+++ b/media/mtransport/nr_socket_prsock.cpp	Tue Sep 17 18:19:04 2019 +0000
@@ -1642,8 +1642,7 @@
     ABORT(R_REJECTED);
   }
 
-  if (config && config->GetProxyPolicy() == NrSocketProxyConfig::kForceProxy &&
-      addr->protocol == IPPROTO_UDP) {
+  if (config && config->GetForceProxy() && addr->protocol == IPPROTO_UDP) {
     ABORT(R_REJECTED);
   }
 